assets/js/script.js

```diff
@@ -150,48 +168,65 @@ function render() {
   $('.js-tab').html('')
   $('.js-preview').html('')
   $('.js-tab').removeClass('active')
+  $('.js-tab').parent().removeClass('active')
   $('.js-swatch').removeClass('active')
   $('.js-loading').hide()
 
   // render bag color
-  $('.js-bag').css({backgroundImage: "url("+(window.baseUrl || '')+"assets/img/bags/danny-"+state.bag+".png)"})
 
-  // render letters in bag and preview
+  $('.js-bag').css({ backgroundImage: "url(" + (window.baseUrl || '') + "assets/img/bags/danny-" + state.bag + ".png)" })
+
+  $('.js-tab').each(function() {
+      if ($(this).data('index') != -1) {
+        var ind = $(this).data('index')
+        var src = `${window.baseUrl || ''}assets/img/letters/` + state.letters[ind].toUpperCase() + `-` + state.materials[ind] + `.png`;
+        $(this).css({ background: "url(" + (window.baseUrl || '') + src + ") no-repeat", backgroundSize: "contain", backgroundPosition: "center" })
+
+      } else {
+        $(this).html('All')
+      }
+    })
+    // render letters in bag and preview
   state.letters.split('').forEach(function(l, i) {
-    var letter = Letter({ letter: l, material: state.materials[i], index: i})
+    var letter = Letter({ letter: l, material: state.materials[i], index: i })
     $('.js-loading').show()
     $('.js-letters').append(letter)
     letter[0].onload = function() {
       $('.js-loading').hide()
-      if( state.activeLetter === i ) {
+      if (state.activeLetter === i) {
         $('.js-preview').append(letter.clone())
       }
     }
   })
 
   // show editor palette
-  if( state.editing ) {
+
+  /*
```

Comment:
so one thing you can start doing now that you're using a git workflow is to just outright delete code instead of commenting out. commented code has a tendency to stick around, and is what's called a "code smell"

just delete this for now, and if you find that you need it back then you can always find it in your git history. related to that, git history is a super useful tool, so you'll want to start giving your commits meaningful messages. you can look at some of my commits for examples of commit messages